test/logflare_web/controllers/api/team_controller_test.exs

```diff
@@ -201,4 +203,100 @@ defmodule LogflareWeb.Api.TeamControllerTest do
              |> assert_schema("NotFoundResponse") == %{error: "Not Found"}
     end
   end
+
+  describe "add_member/2" do
+    test "adds an existing user to a team", %{
+      conn: conn,
+      user: user,
+      main_team: main_team
+    } do
+      new_member = insert(:user)
+
+      assert conn
+             |> add_access_token(user, "private")
+             |> post(~p"/api/teams/#{main_team.token}/members", %{email: new_member.email})
+             |> response(204)
+             |> assert_schema("AcceptedResponse") == ""
```

Comment:
should return created user, should also be a 201 since we're creating a team_user